wandering-warriors/main.py

```diff
@@ -28,8 +35,70 @@ class OperationsBar(Widget):
     pass
 
 
-class CuneiformDrawingInput(Widget):
-    pass
+def calculate_points(x1, y1, x2, y2, steps=1):
+    dx = x2 - x1
+    dy = y2 - y1
+    dist = sqrt(dx * dx + dy * dy)
+    if dist < steps:
+        return
+    o = []
+    m = dist / steps
+    for i in range(1, int(m)):
+        mi = i / m
+        lastx = x1 + dx * mi
+        lasty = y1 + dy * mi
+        o.extend([lastx, lasty])
```

Comment:
The same as `o += [lastx, lasty]` which in my opinion looks cleaner. (I mean, it may be slower but yeah)